lms/djangoapps/lti_provider/outcomes.py

```diff
@@ -0,0 +1,199 @@
+"""
+Helper functions for managing interactions with the LTI outcomes service defined
+in LTI v1.1.
+"""
```

Comment:
In general, we like to encourage fat models. Could you move some of these into methods on the relevant models?